test/testUtils/TestSupport.scala

```diff
@@ -78,7 +79,21 @@ trait TestSupport extends UnitSpec with GuiceOneAppPerSuite with BeforeAndAfterE
   implicit val conf: Configuration = app.configuration
   implicit val environment: Environment = app.injector.instanceOf[Environment]
   implicit val appConfig: FrontendAppConfig = app.injector.instanceOf[FrontendAppConfig]
-  implicit val dateService: DateService = app.injector.instanceOf[DateService]
+
+  // Set fixed date for DateService
+  lazy val fixedDate : LocalDate = LocalDate.of(2023, 12, 15)
+  implicit val dateService: DateService = new DateService {
+
+    override def getCurrentDate(isTimeMachineEnabled: Boolean): LocalDate = fixedDate
```

Comment:
❓ same as before, will we update this every year/taxyear? If not it's not really *current*Date. Same for methods below

Reply:
this is in scope of test implementation of the dateService; we are fixing date to be static for the test, so its fine

Reply:
:memo: => getCurrentDate is defined in the main, this is not something we can change;
we just providing value which is required in the scope of tests etc;
Have a look at the implementation in the main ->
/income-tax-view-change-frontend/app/services/DateService.scala

Reply:
gotcha